### PR TITLE
chore(gitignore): add ide specific folders to .gitignore file

### LINKED ISSUE #5093: Bug: .idea folder is not present in .gitignore file
### URL:

https://nodejs.org

### Browser Name:

Arc

### Browser Version:

Chromium Engine Version 110.0.5481.177

### Operation System:

MacOS 13.2.1 (22D68)

### How to reproduce the issue:

I believe including .idea folder (which is automatically created by any JetBrains application if developer is using JetBrains products) in .gitignore file would makes more sense to not to have "unversioned files" section.

Screenshot 2023-03-09 at 16 28 50

### PR DESCRIPTION
**Pull request by EmirBoyaci**
Fixes #5093

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,9 @@ npm-debug.log
 .AppleDouble
 .LSOverride
 
+# JetBrains IDE
```
Comment (AugustinMauroy) [inline .gitignore]:
```suggestion
# IDE's Files
.idea/
.vscode/
```

**Review by AugustinMauroy — APPROVED** [review]
LGTM!

**Comment by Trott** [comment]
I won't block this, but I would vastly prefer people add these kinds of things to their personal `.gitignore` files rather than to individual repositories. Every repository on the planet should not have to add every IDE's specific files (or `.DS_Store` files or....) But *you* should know about *your* IDE (and your operating system, in the case of `.DS_Store`). So you can put it in one place and solve the problem everywhere for yourself.

**Comment by EmirBoyaci** [comment]
> I won't block this, but I would vastly prefer people add these kinds of things to their personal `.gitignore` files rather than to individual repositories. Every repository on the planet should not have to add every IDE's specific files (or `.DS_Store` files or....) But _you_ should know about _your_ IDE (and your operating system, in the case of `.DS_Store`). So you can put it in one place and solve the problem everywhere for yourself.

I understand what you’re mentioning but in the case some ppl had no idea to do that. I have global git config setup in my windows machine but didn’t do that for my mac machine. And to be honest, I forgot global git config. Because, when I did that I never touched again :D 
I am still thinking adding some kind of global folders/files to gitignore would make more sense to make contributing easily.

**Review by ovflowd — CHANGES_REQUESTED** [review]
Sadly I cannot approve this. I appreciate the intent, tho.

**Comment by EmirBoyaci** [comment]
> Sadly I cannot approve this. I appreciate the intent, tho.

I totally understood their comments but I'm on the opposite side :D

**Pull request closed without merging**